exampleSite/content/users/users.json

```diff
@@ -391,5 +391,12 @@
     "tags": [
       "Personal site mainly in Chinese"
     ]
+    {
+    "title": "jundimubarok.com",
+    "url": "https://jundimubarok",
```

Comment:
Hey @jundi14 

After looking at you PR, it seems that you have no fully completed the things necessary so that your site be added to the users section.

1. Add the full URL to your site [https://jundimubrok.com](https://jundimubarok.com/) (cool site btw 😄 ) on `users.json`
2. Create a folder in `exampleSite/content/users` and add your feature image (
and index.md which is found [here](https://github.com/nunocoracao/blowfish/tree/main/exampleSite/content/users).

I've added examples of what these might look like below for step 2


![feature.jpg](https://github.com/nunocoracao/blowfish/assets/88465473/428cf90b-f8bc-4135-b1de-5f95be1fcd50)
```
title: "jundimubarok.com"
tags: [Personal site]
externalUrl: "https://jundimubarok.com"
date: 9947-08-08
showDate: false
showAuthor: false
showReadingTime: false
showEdit: false
showLikes: false
showViews: false
layoutBackgroundHeaderSpace: false
```

Reply:
Hello @jundi14 the change in the users.json is the only change required. @Turmaxx the md files are generated with the json file not created manually.